### PR TITLE
html seat booking task

### PR DESCRIPTION
**Pull request by alstep07**
### html-movie-seat-booking

[Demo](https://alstep07.github.io/movie-seat-booking/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by ses3332002** [comment]
Great work!
But why you set tabindex="0" on H1 and IMG tags?

**Comment by alstep07** [comment]
> Great work!
> But why you set tabindex="0" on H1 and IMG tags?

For screen-readers.

File: submissions/alstep07/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="Description" content="buy a movie ticket online">
+    <title>movie seat booking</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="booking-container">
+        <header class="header">
+            <h1 class="header__title" tabindex="0">Choose your seats</h1>
```
Comment (A-Ostrovnyy) [inline submissions/alstep07/html-movie-seat-booking/index.html]: By default screen readers have navigation by headings, no need to add `tabindex="0"` to headings.

File: submissions/alstep07/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+            <div class="example">
+                <div class="example__item">
+                    <div class="example__icon example__icon--not-available"></div>
+                    <div class="example__hyphen"></div>
+                    <div class="example__desc">N/A</div>
```
Comment (A-Ostrovnyy) [inline submissions/alstep07/html-movie-seat-booking/index.html]: For text content better to use text tags such as `<p>` etc.

File: submissions/alstep07/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+                    <div class="example__hyphen"></div>
+                    <div class="example__desc">A</div>
+                </div>
+            </div>
+        </header>
+
+        <main class="cinema">
+            <section class="cinema__screen">
+                <h2 class="visually-hidden">Screen</h2>
+                <img src="./img/screen.svg" tabindex="0" class="screen__img" alt="screen">
```
Comment (A-Ostrovnyy) [inline submissions/alstep07/html-movie-seat-booking/index.html]: By default screen readers read all added images in html. No need to add `tabindex` attribute to the images.

File: submissions/alstep07/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+            </section>
+            <section class="cinema__hall">
+                <h2 tabindex="0" class="visually-hidden">cinema hall</h2>
```
Comment (A-Ostrovnyy) [inline submissions/alstep07/html-movie-seat-booking/index.html]: Same as at line 16.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.

File: submissions/alstep07/html-movie-seat-booking/index.html (hunk 1 above)
Comment (alstep07) [reply submissions/alstep07/html-movie-seat-booking/index.html]: Thank you for your review! I used google chrome extension "Screen Reader" and it did not read headers, thats why I added tabindex. I will remove it and test in screen reader, wich you recomended in todays lection.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**